content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/README.md

```diff
@@ -37,6 +38,7 @@ The following properties are written to JCR for this Title component and are exp
 1. `./jcr:title` - will store the text of the title to be rendered
 2. `./type` - will store the HTML heading element type which will be used for rendering; if no value is defined, the component will fallback
 to the value defined by the component's policy
+3. `./linkURL` - will allow defining a URL to which the Title will link to.
```

Comment:
"will allow definition of a content page path, external URL or page anchor for linking the title."

Reply:
Done